src/main/java/com/softserve/academy/spaced/repetition/service/CourseCommentService.java

```diff
@@ -6,15 +6,51 @@
 
 import java.util.List;
 
+/**
+ * This interface works with the comments to courses.
+ */
 public interface CourseCommentService {
+    /**
+     * Adds the comment to course or answer to comment if @param parentCommentId is not null.
+     *
+     * @param courseId        must not be {@literal null}.
+     * @param commentText     text of the comment, must not be {null}.
+     * @param parentCommentId id of the comment on which the answer is given.
+     * @return the comment or answer to comment.
+     * @throws NotAuthorisedUserException
+     */
     CourseComment addCommentForCourse(Long courseId, String commentText, Long parentCommentId)
             throws NotAuthorisedUserException;
 
+    /**
+     * Gets the course's comment with the given identifier.
+     *
+     * @param commentId -  must not be {@literal null}.
+     * @return the comment with the given identifier.
+     */
     CourseComment getCommentById(Long commentId);
 
+    /**
+     * Gets all comments of the course with the given identifier.
+     *
+     * @param courseId - course's id, must not be {@literal null}.
+     * @return list of comments
+     */
     List<Comment> getAllCommentsForCourse(Long courseId);
 
+    /**
+     * Updates comment with the given identifier on the new text
```

Comment:
Update text of comment with the given identifier ( maybe )

Reply:
ok your comment is much better